content/rule_book/internationals.md

```diff
@@ -0,0 +1,60 @@
+# International Bible Quizzing
+
+These rules apply to the *International Bible Quizzing* (IBQ) championship meet conducted annually, administered under the oversight of the *Christian and Missionary Alliance* (CMA) *Quizzing Leadership Team* (CQLT).
+
+## Eligibility Requirements
+
+All quizzers for IBQ competition must be 12 to 18 years of age at any point during the quiz season. Each quiz season begins on August 1 and ends on July 31.
```

Comment:
**Alan Yam:** Under eligibility (7.1), it states that a quizzer must be between 12 – 18 years of age.  Does that mean that a university student is allowed to quiz if they fall in this age group?  (yes, we have had first and second year university students wanting to quiz)

Reply:
This is a great question and one that deserves considerable discussion. The original wording we were working with was. 

> All quizzers for International competition must be 12 to 18 years of age by the meet registration date or have been enrolled as a full-time high school student during the current quiz year.  

The word **OR** here creates some grey area. In an attempt to remove that grey area we dropped the wording about high school enrollment.

Reply:
Full-time high school enrollment also creates some grey areas. There are students that are duel enrolled in college and in high school and may be considered part-time for both. 

Again. This deserves quite a bit of conversation and word-smithing. This may also become a large discussion point for future revisions of the rule book.